.travis.yml

```diff
@@ -26,8 +25,7 @@ before_script:
   - composer update
 
 script:
-  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit ideannotator/tests/; fi
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml; fi
+  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit ideannotator/tests/ --coverage-clover=coverage.xml; fi
   - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs --standard=vendor/silverstripe/framework/phpcs.xml.dist ideannotator/src/ ; fi
```

Comment:
We should run phpcs on the tests dir as well. You'll need to namespace the mocks

Reply:
Except for one, on purpose, a non-namespaced test. All others are already properly namespaced.

Reply:
Ok cool, we can add a standards exclusion to that file for the namespace